documentation/docs/getting_started/nodejs.mdx

```diff
@@ -0,0 +1,81 @@
+---
+description: 'Get started with the official IOTA Wallet Library Software Node.js binding.'
+image: /img/logo/wallet_light.png
+keywords:
+
+- Node.js
+- dotenv
+- install
+- npm
+- yarn
+- security
+
+---
+import SafePasswordStorage from '../_admonitions/_warning_safe_password_storage.md'
+
+# Getting Started with Node.js
+
+The [IOTA Wallet Node.js binding](https://www.npmjs.com/package/@iota/wallet) is published
+on [npmjs.com](https://www.npmjs.com/).
+
+<SafePasswordStorage />
+
+## Prerequisites
+
+* [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) to compile the binding.
```

Comment:
That's only required if no prebuilt binary is available and one needs to compile it
But then also the things listed under `Dependencies` in the Rust getting started are required
Maybe just link to the Rust page for all of that?

Reply:
Would something like this work?

```suggestion
* Make sure you have fulfilled the [Rust Getting Started](rust.mdx) requirements to compile the binding.
```

Reply:
Yes, looks good to me :+1: